### ISSUE TITLE
Division by zero when creating a thumbnail

### ISSUE THREAD

**Issue by tbsnap** [python-pillow/Pillow#4624]
Hi,
Sorry if this as already been reported, but when trying to create a thumbnail (80, 80) from an image (550, 5)  I get a divide by zero 

```python
Traceback (most recent call last):
  File "<interactive input>", line 1, in <module>
  File "C:\Python38\lib\site-packages\PIL\Image.py", line 2279, in thumbnail
    y = round_aspect(x / aspect, key=lambda n: abs(aspect - x / n))
  File "C:\Python38\lib\site-packages\PIL\Image.py", line 2272, in round_aspect
    return max(min(math.floor(number), math.ceil(number), key=key), 1)
  File "C:\Python38\lib\site-packages\PIL\Image.py", line 2279, in <lambda>
    y = round_aspect(x / aspect, key=lambda n: abs(aspect - x / n))
ZeroDivisionError: division by zero
```
It looks like the key (n) can be be zero so perhaps the code needs to test for it?

Tim

**Comment by nulano** [comment]
The relevant code is checking whether it is better to round up or down to better match the image's aspect ratio. In this case, because of the dimensions of the image, rounding down gives a size of 0. The outer max is supposed to take care of it, but the division is failing.

Therefore I would modify `round_aspect` as:

```python
        def round_aspect(number, key):
+            if number < 1:
+                return 1
+            return min(math.floor(number), math.ceil(number), key=key)
-            return max(min(math.floor(number), math.ceil(number), key=key), 1)
```

**Comment by radarhere** [comment]
I've created my idea for a fix in PR #4625

**Comment by radarhere** [comment]
This has now been fixed, and will be part of Pillow 7.2.0, due out on July 1.
